pkgs/development/python-modules/kantoku/default.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  flit-core,
+  gevent,
+  mock,
+  psutil,
+  pytest-cov-stub,
+  pytestCheckHook,
+  pyyaml,
+  pyzmq,
+  tornado,
+}:
+
+buildPythonPackage rec {
+  pname = "kantoku";
+  version = "0.18.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "bentoml";
+    repo = "kantoku";
+    tag = version;
+    hash = "sha256-pI79B7TDZwL4Jz5e7PDPIf8iIGiwCOKFI2jReUt8UNg=";
+  };
+
+  build-system = [ flit-core ];
+
+  dependencies = [
+    psutil
+    pyzmq
+    tornado
+  ];
+
+  nativeBuildInputs = [
+    gevent
+    mock
+    pytest-cov-stub
+    pytestCheckHook
+    pyyaml
+  ];
+
```

```suggestion
    pyyaml
  ];

  nativeCheckInputs = [ pytestCheckHook ];
```

Please use the test runner with the `nativeCheckInputs` attribute.